source/castUtils.ts

```diff
@@ -1,77 +1,78 @@
-import { IGeoLocation, CastColumnTypeEnum } from "./types";
```

Comment:
Reformatting and type improvements. This will be deprecated in the next version.